contrib/lib/tests/databases.rs

```diff
@@ -11,3 +11,65 @@ mod databases_tests {
     #[database("bar")]
     struct PrimaryDb(diesel::PgConnection);
 }
+
+#[cfg(all(feature = "databases", feature = "sqlite_pool"))]
+#[cfg(test)]
+mod rusqlite_integration_test {
+    use std::collections::BTreeMap;
+    use rocket::config::{Config, Environment, Value};
+    use rocket_contrib::databases::rusqlite;
+    use rocket_contrib::database;
+
+    #[database("test_db")]
+    struct SqliteDb(pub rusqlite::Connection);
+
+    #[test]
+    fn deref_mut_impl_present() {
+        let mut test_db: BTreeMap<String, Value> = BTreeMap::new();
+        let mut test_db_opts: BTreeMap<String, Value> = BTreeMap::new();
+        test_db_opts.insert("url".into(), Value::String(":memory:".into()));
```

Comment:
Is this magic `:memory:` URL documented anywhere?

Reply:
It's originally documented at https://sqlite.org/c3ref/open.html, I believe. Neither `rusqlite` nor `r2d2_sqlite` documents this.

Reply:
Okay. Would be nice to let users know, but perhaps it's fine to delegate to lower level docs here.